br/tests/lightning_duplicate_resolution_replace_multiple_keys_clustered_pk/run.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Copyright 2021 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+check_cluster_version 5 2 0 'duplicate detection' || exit 0
+
+run_lightning
+
+# Ensure all tables are consistent.
+run_sql 'admin check table dup_resolve.a'
+
+run_sql 'select count(*) from dup_resolve.a'
+check_contains 'count(*): 5'
+
+run_sql 'select * from dup_resolve.a'
+check_contains 'a: 1'
+check_contains 'b: 1'
+check_contains 'c: 1'
+check_contains 'd: 1.csv'
```

Comment:
more like `ignore` semantic in this case

Reply:
I did not quite understand. Here `1,1,1,1.csv` and `1,1,2,2.csv` have conflicts due to PK, and here in `replace` semantic we reserve the first row and delete the second row.

Reply:
`ignore` will keep the first row met, like what's done here, `replace` will keep the last

Reply:
Here `batch-size = 1`, the actual import order is not the same as that presented in `dup_resolve.a.1`